spec/models/article_spec.rb

```diff
@@ -8,12 +8,16 @@
 
   describe 'Factory' do
     it 'should have valid Factory' do
-      expect(create(:article)).to be_valid
+      expect(create(:user)).to be_valid
```

Comment:
This should be article. You are testing the article factory :) 
If you are getting an error maybe you forgot to add user to the factory?

Reply:
Looked at the article factory and it's missing the user. :)